sources/packages/backend/apps/api/src/route-controllers/program-year/program-year.students.controller.ts

```diff
@@ -6,25 +6,33 @@ import {
   ParseIntPipe,
 } from "@nestjs/common";
 import { ProgramYearService } from "../../services";
-import { OptionItem } from "../../types";
+import { ClientTypeBaseRoute } from "../../types";
 import BaseController from "../BaseController";
 import { AllowAuthorizedParty } from "../../auth/decorators/authorized-party.decorator";
 import { AuthorizedParties } from "../../auth/authorized-parties.enum";
-import { ProgramYearDto } from "./models/program-year.dto";
-import { ApiTags } from "@nestjs/swagger";
+import { ProgramYearAPIOutDTO } from "./models/program-year.dto";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
 import { RequiresStudentAccount } from "../../auth/decorators";
+import { OptionItemAPIOutDTO } from "../models/common.dto";
 
 @AllowAuthorizedParty(AuthorizedParties.student)
 @RequiresStudentAccount()
 @Controller("program-year")
-@ApiTags("program-year")
+@ApiTags(`${ClientTypeBaseRoute.Student}-program-year`)
 export class ProgramYearController extends BaseController {
   constructor(private readonly programYearService: ProgramYearService) {
     super();
   }
 
+  /**
+   * Gets a list of program years returned as option items (id/description pair).
+   * @returns an array of program years as OptionItemAPIOutDTO.
```

Comment:
Probably an old code but we try to avoid adding the return type here `OptionItemAPIOutDTO`.

Reply:
Shouldn't we use the generic common `OptionItemAPIOutDTO` and create a specific "OptionItem" for every options list?

Reply:
My comment was only about the "comment" (**line 29**). In the comments, we decided to try to avoid adding the type already explicit in the method signature because we end up renaming the types and usually forgot the type. 

![image](https://user-images.githubusercontent.com/61259237/208488881-c2d3fad5-8b38-4e90-abd0-2e18b8e180ac.png)

Reply:
Oh, got it. Thanks!